apps/web/app/actions.ts

```
@@ -2,118 +2,200 @@
+
+		if (callbackUrl) {
+			response.redirect = callbackUrl
+		}
 
-	return encodedRedirect(
-		'success',
-		'/forgot-password',
-		'Check your email for a link to reset your password.',
-	)
+		return response
+	} catch (error: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)